src/moments.jl

```diff
@@ -58,99 +84,184 @@ function var!(R::AbstractArray, A::RealArray, wv::WeightVec, dim::Int; mean=noth
 """
-Base.stdm(v::RealArray, wv::WeightVec, m::Real) = sqrt(varm(v, wv, m))
+Base.stdm(v::RealArray, wv::AbstractWeights, m::Real; corrected::DepBool=nothing) =
+    sqrt(varm(v, wv, m, corrected=depcheck(:stdm, corrected)))
 
 """
-    std(v, wv::WeightVec, [dim]; mean=nothing)
+    std(v, wv::AbstractWeights, [dim]; mean=nothing, corrected=false)
+
+Compute the standard deviation of a real-valued array `x`,
+optionally over a dimension `dim`. Observations in `x` are weighted using weight vector `w`.
 
-Return the standard deviation of a real-valued array `v`, optionally over a
-dimension `dim`. The weighting vector `wv` specifies frequency weights (also
-called case weights) for the estimate.
+The uncorrected (when `corrected=false`) sample standard deviation is defined as:
+
+```math
+\\sqrt{\\frac{1}{\\sum{w}} \\sum{i=1}^n {w_i\\left({x_i - μ}\\right)^2 }}
+```
+where ``n`` is the length of the input and ``μ`` is the mean.
+
+The unbiased estimate of the population standard deviation is computed by replacing
+``\\frac{1}{\\sum{w}}`` with a factor dependent on the type of weights used:
+
+* AnalyticWeights: ``\\frac{1}{\\sum w - \\sum {w^2} / \\sum w}``
+* FrequencyWeights: ``\\frac{1}{\\sum{w} - 1}``
+* ProbabilityWeights: ``\\frac{n}{(n - 1) \\sum w}`` where ``n`` equals `count(!iszero, w)`
 """
-Base.std(v::RealArray, wv::WeightVec; mean=nothing) = sqrt.(var(v, wv; mean=mean))
+Base.std(v::RealArray, wv::AbstractWeights; mean=nothing, corrected::DepBool=nothing) =
+    sqrt.(var(v, wv; mean=mean, corrected=depcheck(:std, corrected)))
 
-Base.stdm(v::RealArray, m::RealArray, dim::Int) = Base.sqrt!(varm(v, m, dim))
-Base.stdm(v::RealArray, wv::WeightVec, m::RealArray, dim::Int) = sqrt.(varm(v, wv, m, dim))
-Base.std(v::RealArray, wv::WeightVec, dim::Int; mean=nothing) = sqrt.(var(v, wv, dim; mean=mean))
+Base.stdm(v::RealArray, m::RealArray, dim::Int; corrected::DepBool=nothing) =
+    Base.sqrt!(varm(v, m, dim; corrected=corrected))
 
+Base.stdm(v::RealArray, wv::AbstractWeights, m::RealArray, dim::Int;
+          corrected::DepBool=nothing) =
+    sqrt.(varm(v, wv, m, dim; corrected=depcheck(:stdm, corrected)))
+
+Base.std(v::RealArray, wv::AbstractWeights, dim::Int; mean=nothing,
+         corrected::DepBool=nothing) =
+    sqrt.(var(v, wv, dim; mean=mean, corrected=depcheck(:std, corrected)))
 
 ##### Fused statistics
 """
-    mean_and_var(x, [wv::WeightVec], [dim]) -> (mean, var)
+    mean_and_var(x, [wv::AbstractWeights], [dim]; corrected=false) -> (mean, var)
 
 Return the mean and variance of a real-valued array `x`, optionally over a dimension
-`dim`, as a tuple. A weighting vector `wv` can be specified to weight the estimates.
-The weights are assumed to be frequency weights, also called case weights.
+`dim`, as a tuple. Observations in `x` can be weighted via `wv`. Finally, bias correction
+can be applied to the variance calculation if `corrected=true`.
```

Comment:
"can be" -> "is/will be" (as noted in a previous comment -- there are probably several occurrences of this)